apps/etterlatte-behandling/src/main/kotlin/behandling/regulering/ReguleringAggregat.kt

```diff
@@ -0,0 +1,76 @@
+package no.nav.etterlatte.behandling.regulering
+
+import no.nav.etterlatte.behandling.BehandlingDao
+import no.nav.etterlatte.behandling.domain.Regulering
+import no.nav.etterlatte.behandling.foerstegangsbehandling.FoerstegangsbehandlingAggregat
+import no.nav.etterlatte.behandling.hendelse.HendelseDao
+import no.nav.etterlatte.behandling.hendelse.HendelseType
+import no.nav.etterlatte.behandling.hendelse.registrerVedtakHendelseFelles
+import no.nav.etterlatte.libs.common.behandling.BehandlingStatus
+import no.nav.etterlatte.libs.common.behandling.BehandlingType
+import no.nav.etterlatte.libs.common.behandling.Persongalleri
+import no.nav.etterlatte.libs.common.tidspunkt.Tidspunkt
+import org.slf4j.LoggerFactory
+import java.time.LocalDateTime
+import java.util.*
+
+class ReguleringAggregat(
+    id: UUID,
+    behandlinger: BehandlingDao,
+    private val hendelser: HendelseDao
+) {
+    companion object {
+        private val logger = LoggerFactory.getLogger(FoerstegangsbehandlingAggregat::class.java)
+
+        fun opprettRegulering(
+            sak: Long,
+            persongalleri: Persongalleri,
+            behandlinger: BehandlingDao,
+            hendelser: HendelseDao
+        ): ReguleringAggregat {
+            logger.info("Oppretter en behandling på $sak")
+            return Regulering(
+                id = UUID.randomUUID(),
+                sak = sak,
+                behandlingOpprettet = LocalDateTime.now(),
+                sistEndret = LocalDateTime.now(),
+                status = BehandlingStatus.OPPRETTET,
+                persongalleri = persongalleri,
+                kommerBarnetTilgode = null,
+                vilkaarUtfall = null,
+                virkningstidspunkt = null
```

Comment:
Disse burde nok ikke være `null` men heller få verdien fra `forrigeBehandling` sånn som persongalleri

Reply:
Virkningstidspunkt burde bestemmes fra datoen som sendes med på kafka-meldingen

Reply:
Kokte sånn vi gjer det for revurdering, men det bilr kanskje litt annleis her ja.